### PR TITLE
feat: add links to Remix with the Starknet plugin enabled and the file

### LINKED ISSUE #60: feat: Remix cairo integration
Now that the `Remix Starknet plugin` has been released, we can consider linking each example to a remix workspace with the corresponding contracts. This will enable readers to experiment directly with the contract in their browser.

### PR DESCRIPTION
**Pull request by EdgarBarrantes**
Closes #60

### CODE REVIEW

File: src/ch02-01-upgradeable_contract.md [hunk 1]
```diff
@@ -25,9 +25,11 @@ Start by deploying `UpgradeableContract_V0` as the initial version. Next, send a
...
 ```
-Visit contract on [Voyager](https://goerli.voyager.online/contract/0x005300003ade5d10447d941a42d48b7141074cd8bade2b16520684896a5090ea).
+Visit contract on [Voyager](https://goerli.voyager.online/contract/0x005300003ade5d10447d941a42d48b7141074cd8bade2b16520684896a5090ea) or play with it in [Remix](https://remix.ethereum.org/?#activate=Starknet-cairo1-compiler&url=https://github.com/NethermindEth/StarknetByExample/blob/main/listings/ch02-applications/upgradeable_contract/src/upgradeable_contract_v0.cairo).
+
 
 ```rust
 {{#include ../listings/ch02-applications/upgradeable_contract/src/upgradeable_contract_v1.cairo}}
 ```
-Visit contract on [Voyager](https://goerli.voyager.online/contract/0x017c86152badd1d665b9836571bd6b0a484f028748aa13d9b2d5d9c9192fafc6).
+Visit contract on [Voyager](https://goerli.voyager.online/contract/0x017c86152badd1d665b9836571bd6b0a484f028748aa13d9b2d5d9c9192fafc6) or play with it in [Remix](https://remix.ethereum.org/?#activate=Starknet-cairo1-compiler&url=https://github.com/NethermindEth/StarknetByExample/blob/main/listings/ch02-applications/upgradeable_contract/src/upgradeable_contract_v0.cairo).
```
Comment (julio4) [inline src/ch02-01-upgradeable_contract.md:34]: upgradeable_contract_v**1**.cairo

**Review by julio4 — CHANGES_REQUESTED** [review]
LGTM, just one url to fix.

**Review by julio4 — APPROVED** [review]
Perfect! Thank you!

**Comment by julio4** [comment]
CI should be fixed after #66

**Comment by enitrat** [comment]
Did you tweet? 🔥

**Comment by julio4** [comment]
@enitrat Not yet, CI is broken so it's not deployed yet. Waiting for #66 (if you can approve by the way)

**Pull request merged**